app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/view_return.scala.html

```diff
@@ -36,7 +38,7 @@
         lossesSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.returns.exemptionandlosses.summary
 )
 
-@(completeReturn: CompleteReturn, returnSummary: ReturnSummary)(implicit request: RequestWithSessionData[_], messages:Messages, viewConfig: ViewConfig)
+@(completeReturn: CompleteReturn, returnSummary: ReturnSummary, cutoffDate: LocalDate, isUk: Boolean, isRebasing: Boolean)(implicit request: RequestWithSessionData[_], messages:Messages, viewConfig: ViewConfig)
```

Comment:
something like `rebasingCutOff`date instead of `cutOff` date would be more descriptive here and in other places